cmd/seccompagent/seccompagent.go

```diff
@@ -117,13 +117,20 @@ func main() {
 
 			r := registry.New()
 
+			// DEFAULT_ACTION=kill-container differs from SCMP_ACT_KILL_PROCESS that
+			// it kills the pid1 of the container, terminating all processes of the
+			// container instead of just the process making the syscall.
+			if v, ok := metadata["DEFAULT_ACTION"]; ok && v == "kill-container" {
+				r.DefaultHandler = handlers.KillContainer(podCtx.Pid1)
+			}
```

Comment:
UX: I'd warn if `v` is not known value, so the user knows they have a typo, they are using an old version that doesn't support that action, etc.

Reply:
I implemented the error message in seccomp-agent's logs

Reply:
Oh, yeah, a log msg is exactly what I meant :D